### PR TITLE
Review corrections

### PR DESCRIPTION
**Pull request by le-doude**
Fix stuff pointed out by first master code review

### CODE REVIEW

**Comment by pcboy** [comment]
src/test/scala/io/enjapan/Fuzzing.scala

Either this thing should be done generically and be put inside its own library (we could use it in other projects), or you should simply use a testing lib supporting fuzzying (scalatest?).

File: src/main/scala/io/enjapan/ops/TypedOperations.scala [hunk 1]
```diff
@@ -73,6 +73,14 @@ abstract class AsyncTypedOpsImpl extends TypedOperations[Future] {
...
 abstract class BlockingTypedOpsImpl extends TypedOperations[Try] {
   self: JavaBlockingBucketProvider =>
+  /**
+   * Making RawJsonDocument is faster as it is the same as what the Java SDK is doing in the end
+   * With RawJsonDoc => CaseClass -> JsValue -> String -> ByteArray
+   * With JsonObjectDoc => CaseClass -> JsValue -> JsonDocument -> String -> ByteArray
+   *
+   * Both end up checking the Json format is valid anyway.
+   */
+
```
Comment (shawila) [inline src/main/scala/io/enjapan/ops/TypedOperations.scala]: Is the SDK slow at doing so?

File: src/main/scala/io/enjapan/utils/Conversions.scala [hunk 2]
```diff
@@ -11,13 +12,29 @@ import scala.util.Try
...
+        case string: String => JsString(string)
+        case jsArray: JsonArray => JsArray(jsArray.asScala.toSeq.map({ v => innerRecursiveConversion(v)}))
+        case jsObject: JsonObject => JsObject(jsObject.toMap.asScala.toSeq.map({
+          case (k, v) => k -> innerRecursiveConversion(v)
+        }))
+        case _ => JsNull
+      }
+    }
+    innerRecursiveConversion(jsonObject)
+  }
```
Comment (shawila) [inline src/main/scala/io/enjapan/utils/Conversions.scala]: Porque Maria?

File: src/main/scala/io/enjapan/ops/TypedOperations.scala (hunk 1 above)
Comment (le-doude) [inline src/main/scala/io/enjapan/ops/TypedOperations.scala]: It is always jackson json in the background. It is just one extra computation for nothing.

**Comment by shawila** [comment]
LGTM!
[LGTM](http://www.lgtm.in/i/25EknQNlq)

[:+1:](http://www.lgtm.in/u/25EknQNlq) [:-1:](http://www.lgtm.in/r/25EknQNlq)

**Comment by pcboy** [comment]
Far less code than before, that's a success!

LGTM

**Pull request merged**